addons/l10n_cn_standard/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
+    Copyright (C) 2008-2008 凯源吕鑫 lvxin@gmail.com   <basic chart data>
+    Copyright (C) 2008-now  开阖软件 jeff@osbzr.com    < PM and LTS >
+    Copyright (C) 2017-now  jeffery9@gmail.com    
```

Comment:
please avoid putting copyright notice in every file, you can put it in the description of the module if needed